apps/andi/test/unit/andi_web/live/dataset_live_view/table_test.exs

```diff
@@ -154,7 +154,7 @@ defmodule AndiWeb.DatasetLiveViewTest.TableTest do
 
     {:ok, _view, html} = live(conn, @url_path)
 
-    assert get_rendered_table_cells(html) == [["", dataset.business.dataTitle, dataset.business.orgTitle]]
+    assert get_rendered_table_cells(html) == [["", dataset.business.dataTitle, dataset.business.orgTitle, "Edit"]]
```

Comment:
Maybe I'm missing it, but is there a test verifying the href of the "Edit" link?

Reply:
No, it seemed like that fell under that category of things not worth testing in the UI. We added it because the tests failed otherwise. I think that's a smell and thought about addressing it in a better way but we decided the JSR wasn't there. Do you feel differently?

Reply:
This feels like something that I *think* would be easy to test and documents exactly what the expected path should be.  Seems like that's enough juice for very not much squeeze.  But that's just this juicer's opinion.

Reply:
If I were working the card I would have agreed with Nathaniel... but now that Tony's called it out I can see it making sense... 🤷‍♂

Reply:
Cool, we'll do that.